server/app.js

```diff
@@ -176,13 +175,20 @@ const createApp = ({
     ),
   );
 
-  // GovUK Template Configuration
-  const establishmentId = getEstablishmentId(config.establishmentName);
-  app.locals.asset_path = '/public/';
-  app.locals.config = {
-    ...config,
-    establishmentId,
-  };
+  app.use((req, res, next) => {
+    const replaceUrl = /-prisoner-content-hub.*$/g;
```

Comment:
This string could be put into config I guess as you can do `new RegExp` from a string.

Reply:
This looks great, and would work nicely for review apps as well if we sit them under a domain for each PR/deployment 🙌